import "dos.proto";
package dos;

option cc_generic_services = true;
option py_generic_services = true;

message SubmitJobRequest{
    optional string uid = 1;
    optional JobSpec job = 2;
}

message SubmitJobResponse {
    optional Status status = 1; 
}

service Master {

    // submit a job to master 
    rpc SubmitJob(SubmitJobRequest) returns(SubmitJobResponse);
}
